starter-core/src/main/java/io/micronaut/starter/feature/TestTemplateProvider.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2020 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.starter.feature;
+
+import com.fizzed.rocker.RockerModel;
+import io.micronaut.starter.application.Project;
+import io.micronaut.starter.application.generator.GeneratorContext;
+import io.micronaut.starter.options.Language;
+import io.micronaut.starter.options.TestFramework;
+
+public interface TestTemplateProvider {
```

Comment:
Same thing why the need for an interface? These are not going to be called outside the feature. Please delete this.

Reply:
I am using this in AwsLambda Feature and also in AzureFunction feature and I think they will be used in many other features. Do you still want me to remove them?

Reply:
I've deleted them.